src/components/layout/TaskListSidebar.tsx

```
@@ -0,0 +1,268 @@
+
+    const getListIcon = (list: TaskList) => {
+        switch (list.icon) {
+            case 'work':
+                return <Briefcase className="h-4 w-4 mr-2" />;
+            case 'personal':
+                return <CheckCircle2 className="h-4 w-4 mr-2" />;
+            case 'home':
+                return <Home className="h-4 w-4 mr-2" />;
+            case 'study':
+                return <BookOpen className="h-4 w-4 mr-2" />;
+            case 'default':
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)